src/testUtils.js

```diff
@@ -2,10 +2,10 @@ const path = require('path')
 const fs = require('fs')
 const { promisify } = require('util')
 const execa = require('execa')
-const td = require('testdouble')
 
 const writeFile = promisify(fs.writeFile)
 
+// eslint-disable-next-line no-multi-assign
```

Comment:
You can change this with the following and drop the ESLint disable (can't suggest change because it exceeds the changed line +3).

```js
const updateEnvVars = envVars => {
  Object.keys(envVars).forEach(key => {
    process.env[key] = envVars[key]
  })
}
exports.updateEnvVars = updateEnvVars
```